templates/sectionsummary.html

```diff
@@ -26,18 +26,19 @@ <h1>{{content.summary.title}}</h1>
     <h1>{{content.summary.title}}</h1>
     {% for summary in content.summary.custom_summary %}
       {% if summary.type == 'List' %}
-        <div data-qa="{{summary.list_name}}-list-summary">
+
+          <div {{'class="u-mb-l"' if not loop.last }} data-qa="{{summary.list_name}}-list-summary">
           <h2>{{summary.title}}</h2>
           {% if summary.list and summary.list.list_items %}
             {% set list = summary.list %}
             {% include 'partials/summary/list-summary.html' %}
           {% elif summary.empty_list_text %}
-            <div class="u-mb-s">
+            <div>
               {{ summary.empty_list_text }}
             </div>
           {% endif %}
           {% if summary.add_link %}
-            <div class="u-mb-l">
+            <div>
```

Comment:
The add item link looks closer to the last list item than on the screenshot attached to the Trello card.

Reply:
@jrbarnes9 ?

Reply:
The screen shot is irrelevant now, because we found we couldn't control the border and spacing between the summary items and what follows.  So we've got a new card in DS to bring these add links into the summary component